TWLight/templates/new_base.html

```diff
@@ -10,6 +10,26 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
 
+    <meta property="og:title" content="The Wikipedia Library" />
```

Comment:
I missed this before, but this should be localized as well.
```suggestion
      {% comment %}Translators: Name of the project/tool.{% endcomment %}
    <meta property="og:title" content="{% trans "The Wikipedia Library" %}" />
```

Reply:
Oh, I missed this as well, thank you for pointing it out. I have added the trans tag in the latest commit